rmc/utils/mpc.py

```diff
@@ -406,6 +425,20 @@ def run_regressions(
     ht = ht.transmute(polyphen=ht.polyphen.score)
     df = ht.to_pandas()
 
+    # NOTE: `Table.to_pandas()` outputs a DataFrame with nullable dtypes
+    # for int and float fields, which patsy cannot handle
+    # These must be converted to non-nullable standard numpy dtypes
+    # prior to working with patsy
```

Comment:
```suggestion
    # prior to working with patsy
    # See: https://github.com/hail-is/hail/commit/da557655ef1da99ddd1887bd32b33f4b5adcec9f
```
(just adding more context for future reference)